charts/iter8/templates/_payload-github.tpl

```diff
@@ -0,0 +1,4 @@
+{
+  "event_type": "iter8",
+  "client_payload": {{ .JSONStringReport | indent 2 | trim}}
```

Comment:
Instead of `JSONStringReport`, we could instead try `. | toJson` (or prettyJson, or rawJson). These funcs are already part of the current provider template func map.